src/main/java/FileResourceHandler.java

```diff
@@ -26,11 +28,23 @@ public boolean delete(String fileName) {
         return resource.delete();
     }
 
+    @Override
+    public byte[] read(String filename) {
+        try {
+            System.out.println("Looking up resource at location: " + fullPath(filename));
+            return Files.readAllBytes(Paths.get(fullPath(filename)));
+        } catch (IOException e) {
+            System.out.println("FILE IS NOT Found!!!");
+            return noResourceContentAvailable();
+        }
+    }
+    private byte[] noResourceContentAvailable() {
+        return null;
```

Comment:
What about returning an empty byte array? I worry about the use of `null` in general as it can cause a lot of trouble down the line. Also, my limited brain struggles with the concept of *nothing* in the codebase ;)